### ISSUE TITLE
Need to be able to view edge config, including twin data from iotedgectl

### ISSUE THREAD

**Issue by jonbgallant** [Azure/iot-edge#434]
I would like a dump of the config and twin data, device and module twin, from iotedgectl.

Something like:
```
iotedgectl status --verbose
```

That way we can see exactly what config the runtime is using to assist in debugging.

**Comment by varunpuranik** [comment]
Yes, it would probably be useful to be able to view the iotedgectl config - we will add a work item to our backlog to add that capability.

However, the other things like module twins, etc. are outside the realm of iotedgectl. Work is planned to enable these capabilities on the azure cli (which is the right place for it).
